students/uss_tomasz/lesson_08_regular_expressions/8_3/email_validator.py

```diff
@@ -0,0 +1,22 @@
+# write email validator that checks if supplied string is valid e-mail address
+import re
+
+
+def is_email(user_email):
+    email_regex = re.compile(r'''(
+        [a-zA-Z0-9._%+-]+  
+        @
+        [a-zA-Z0-9.-]+  
+        .
+        \.[a-zA-Z]{2,4}
+        )''', re.VERBOSE)
+    result = re.findall(email_regex, user_email)
+    if len(result) == 0:
+        return False
+    else:
+        return True
```

Comment:
It's a bit overcomplicated. You can just check

```return len(result) > 0```

or even like this using match

```return re.match(email_regex, user_email) is not None```

Same applies to other places in this PR where you used this design pattern